package.json

```diff
@@ -26,6 +26,7 @@
     "themis": "^1.1.4",
     "tv4": "^1.1.9",
     "z-schema": "^3.3.2",
+    "schemasaurus": "0.7.3",
```

Comment:
I prefer to use `^0.7.3`, so non-breaking changes are included when npm install is run.